test/builtin_dataset_mocks.py

```diff
@@ -911,6 +911,61 @@ def country211(info, root, config):
     return num_examples * len(classes)
 
 
+@register_mock
+def food101(info, root, config):
+    data_folder = root / "food101"
+
+    num_images_per_class = 3
+    image_folder = data_folder / "images"
+    categories = ["apple_pie", "baby_back_ribs", "baklava"]
```

Comment:
It is always good practice to include the first and last category to avoid problems for these "extreme" cases.

```suggestion
    categories = ["apple_pie", "baby_back_ribs", "waffles"]
```